docs/aws-professional/services.md

```diff
@@ -7,13 +7,13 @@ ms.date: 03/24/2017
 
 pnp.series.title: Azure for AWS Professionals
 ---
-# AWS to Azure services comparison
+# Azure & AWS Cloud Service Map
 
-This article helps you understand how Microsoft Azure services compare to Amazon Web Services (AWS). Whether you are planning a multicloud solution with Azure and AWS, or migrating to Azure, you can compare the IT capabilities of Azure and AWS services in all categories.
+This document defines how common cloud services are made available via Azure and Amazon Web Services (AWS). We hope that this guide helps developers understand the services that Azure offers, whether they are new to the cloud or just new to Azure. We’ve structured the guide using a table that explains each cloud service capability sorted by service popularity, and maps the capability to the service offering name for both Azure and AWS. In the following tables, there are multiple Azure services listed for some AWS services. The Azure services are similar to one another, but depth and breadth of capabilities vary.
 
-In the following tables, there are multiple Azure services listed for some AWS services. The Azure services are similar to one another, but depth and breadth of capabilities vary.
+[Downloada PDF of the Azure & AWS Cloud Service Map](https://aka.ms/awsazureguide)
```

Comment:
Right now this link goes to `https://www.microsoft.com/en-us/`

Reply:
need space between "download a"

Reply:
The PDF was just updated on MS Download. It works properly now.